src/parser.ml

```diff
@@ -440,16 +440,18 @@ let require : (Files.module_path -> unit) Pervasives.ref = ref (fun _ -> ())
 
 (** [cmd] is a parser for a single command. *)
 let parser cmd =
-  | _require_ m:{_open_ -> P_require_open}?[P_require_default] p:path
```

Comment:
Rather use something like `_require_ o:{_open_ -> true}[false] ps:path+` to combine `require` and `require open` as suggested in my comment on `src/syntax.ml`.